src/util/deleteThought.js

```diff
@@ -66,10 +66,21 @@ export const deleteThought = () => {
     ? prevContext()
     : prevSibling(value, context, rank)
 
-  const next = perma(() =>
-    showContexts
-      ? unroot(getContextsSortedAndRanked(headValue(contextOf(path))))[0]
-      : sortPreference === 'Alphabetical' ? getThoughtsSorted(context)[1] : getThoughtsRanked(context)[0]
+  const next = perma(() => {
+    if (showContexts) {
+      return unroot(getContextsSortedAndRanked(headValue(contextOf(path))))[0]
+    }
```

Comment:
The multi-line ternary is more compact. Please follow existing project conventions where possible.

Reply:
Got it.